.github/ci/integration_test.sh

```diff
@@ -140,7 +143,7 @@ bazel_ci test \
   --test_output=streamed \
   --test_tag_filters="external" \
   --strategy=TestRunner=standalone \
-  //... //src/go/tests:go_default_test //src/go/tests:relay_test
+  //...
```

Comment:
Did you check this still runs k8s_integration_test.go? I thought that `manual` means it's not matched by //...
https://github.com/googlecloudrobotics/core/blob/e5d771c1e7060198f2fb6d1ea5a7f5740efe181f/src/go/tests/BUILD.bazel#L33

Reply:
It does, so my understand is faulty, not the script:

```
//src/go/tests:go_default_test                                           PASSED in 11.3s
//src/go/tests:relay_test                                                PASSED in 10.8s
```

Reply:
See above.